components/infobox/extensions/commons/infobox_extension_game_appearances.lua

```diff
@@ -25,18 +26,20 @@ function Appearances.player(args)
 	end)
 	table.insert(conditions, '[[opponentname::' .. args.player .. ']]')
 
-	local data = mw.ext.LiquipediaDB.lpdb('placement', {
+	local queriedGames = Array.map(mw.ext.LiquipediaDB.lpdb('placement', {
 		conditions = table.concat(conditions, ' OR '),
 		query = 'game',
 		groupby = 'game asc',
 		limit = 1000,
-	})
+	}), function(item) return item.game end)
 
-	local games = Array.unique(Array.map(data, function(item)
-		return Game.name{game = item.game}
-	end))
-	table.sort(games)
-	return games
+	local orderedGames = Array.filter(Game.listGames{ordered = true}, function(game)
+		return Table.includes(queriedGames, game)
```

Comment:
I'm not sure if the return from placement.game is garantreed to be the same as the value from listGames
cc @hjpalpha @mbergen

Reply:
it should be iirc
if we want to be sure we can add a `Game.toIdentifier`